config/boards/orangepizero3.wip

```diff
@@ -3,6 +3,7 @@ BOARD_NAME="Orange Pi Zero3"
 BOARDFAMILY="sun50iw9"
 BOARD_MAINTAINER="viraniac"
 BOOTCONFIG="orangepi_zero3_defconfig"
+OVERLAY_PREFIX=sun50i-h616
```

Comment:
@stephengraf, just from a consistency perspective, should this value be quoted like the others?